src/api/apollo/client.ts

```diff
@@ -1,19 +1,23 @@
+import { InMemoryCache } from 'apollo-cache-inmemory';
 import { ApolloClient } from 'apollo-client';
 import { ApolloLink } from 'apollo-link';
-import { InMemoryCache } from 'apollo-cache-inmemory';
+import DebounceLink from 'apollo-link-debounce';
 import { HttpLink } from 'apollo-link-http';
+import SerializingLink from 'apollo-link-serialize';
 import fetch from 'isomorphic-fetch';
-import localResolvers from './resolvers';
-
 import { ResultType } from './generated/graphql';
+import localResolvers from './resolvers';
 
 const PLAYGROUND_API = process.env.PLAYGROUND_API;
+const DEFAULT_DEBOUNCE_TIMEOUT = 500;
```

Comment:
Should we increase the debounce time to at least 1 sec? I tested out and my typing is around 5 character / second, I feel the average must be a bit lower, so if it's half then we only save one request. I feel we should be dropping much more.

Reply:
@sideninja sure, I'll increase it to 1200ms